package.json

```diff
@@ -39,7 +39,8 @@
   "types": "lib/index.d.ts",
   "dependencies": {
     "@opensea/seaport-js": "^4.0.0",
-    "ethers": "^6.9.0"
+    "ethers": "^6.9.0",
+    "opensea-js": "^7.1.13"
```

Comment:
why is this added?

Reply:
based on https://www.notion.so/opensea/EPD-Supporting-a-new-chain-7b4f81c6dd26413eb784db78f181afd0?pvs=4#7a76b961bef74121bf6bfe2b9c086f76

Reply:
need to bump the version above. add to deps here